docs/developer/mesh.md

```diff
@@ -0,0 +1,270 @@
+# Mesh
+
+Here we describe the two mesh formarts that are used in RDycore. The two mesh
+formats that we use are:
+
+1. Exodus II
+2. DMPlex HDF5 v3.0.0
+
+RDycore has been tested for meshes that include triangular, quadrilateral, or
+both triangular and quadrilateral cells. The vertices of cells have coordinates
+in 3D.
+
+## Exodus II Mesh Format
+
+The Exodus II mesh format (`.exo`) uses 1-based indices.
+
+### Triangular elements
+
+```text
+     Triangular
+        cell
+         v3
+         / \
+        /   \
+      e5     e4
+     /   c1   \
+    /          \
+  v1 --- e3 --- v2
+```
+
+- The `elem_type` of the triangular cell, `c1`, is `TRI3`
+- It comprises of three vertices (i.e. `v1`, `v2`, `v3`)
```

Comment:
"To comprise" is literally "to consist of", so this reads "It consists of of three vertices." Almost no one knows how to use the word "comprise" correctly, so I usually suggest avoiding it. :-)

For the first two bullets, how about

* The triangular cell `c1` consists of the three vertices `v1`, `v2`, and `v3`.
* The `elem_type` of `c1` is `TRI3`.

Reply:
Updated as suggested.